scripts/copy_output.lua

```diff
@@ -53,6 +54,11 @@ for _, image in ipairs(lib.images) do
 	end
 end
 
+local kernel_debug_source = string.format('openwrt/bin/targets/%s/kernel-debug.tar.bz2',
+	bindir)
+local kernel_debug_dest   = env.GLUON_IMAGEDIR..'/debug/'..target..'-kernel-debug.tar.bz2'
```

Comment:
Hmm, we should prefix the destination name including the site and release (and clean up old files using a wildcard, as it is done for images and the opkg repo). It should be possible to reuse some of the opkg repo code below.

Reply:
I have updated my branch to the naming scheme you suggested.

For example for  a build with the following make call:
```$ make all GLUON_TARGET=ar71xx-generic V=s -j10 GLUON_RELEASE=v2019.1.1-ffbs-19-g03876975-202004192147-parker_beta GLUON_BRANCH=parker_beta GLUON_PRIORITY=0```

I now the following files:
```
$ ls output/images/factory/gluon-ffbs-next-v2019.1.1-ffbs-19-g03876975-202004192147-parker_beta-* | head -n1
output/images/factory/gluon-ffbs-next-v2019.1.1-ffbs-19-g03876975-202004192147-parker_beta-alfa-network-ap121.bin

$ ls output/images/debug                                                                                                    
ffbs-next-v2019.1.1-ffbs-19-g03876975-202004192147-parker_beta-ar71xx-generic-kernel-debug.tar.bz2
```

Before copying new files I remove all `-kernel-debug.tar.bz2`for the given `target`.
So I you do multiple makes for different architectures you end up with a `kernel-debug.tar.bz2` for every architecture.

Is that what you intended?